lib/ui/views/new_dish/new_dish_viewmodel.dart

```diff
@@ -27,7 +27,6 @@ class NewDishViewModel extends FormViewModel {
       );
 
       final response = await _dishService.createDish(info);
-
       if (response != null) {
```

Comment:
Hmm, actually for this, you need to take care of the null, it's something that can happen and we need to let the user know that `an unknown error has occurred`.  Using the negative first approach to building the success area in a `try-and-catch` block helps solve a lot in the mobile space.

Reply:
💡